devfile.yaml

```diff
@@ -39,7 +42,10 @@ components:
         - name: mongodb
           exposure: internal
           targetPort: 27017
-      memoryLimit: 512Mi
+      memoryLimit: '512Mi'
+      memoryRequest: '256Mi'
+      cpuLimit: '1'
+      cpuRequest: '250m'
```

Comment:
in other places you have set 1 or 0.5; why not have 0.25 here?

Reply:
changed to 0.25